omod/src/main/webapp/fragments/patientdashboard/visitIncludes.gsp

```diff
@@ -73,19 +73,33 @@
             ])}
         </p>
 
-        <p>
-            <label for="stopDate" class="required">
+        <p>          
+                <label for="stopDate" class="required">
                 ${ ui.message("coreapps.stopDate.label") }
-            </label>
+                </label>
 
-            ${ ui.includeFragment("uicommons", "field/datetimepicker", [
+ <% if(activeVisits){ %>
```

Comment:
Any reason for changing the formatting here?

Reply:
Not an intended change...